Alert.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
```

Comment:
You should create different PRs for different components as suggested.

Reply:
Actually I had Created
but i wanted to get for all so I did merge to Development branch